.github/workflows/publish.yaml

```diff
@@ -0,0 +1,138 @@
+name: Publish Python 🐍 distribution 📦 to PyPI and TestPyPI
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+
+    steps:
+      # Step 1: Checkout the first repository
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      # Step 2: Set up Node.js
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20" # Specify the Node.js version to use
+
+      # Step 3: Install openapi-merge-cli and openapi-generator-cli
+      - name: Install openapi-merge-cli
+        run: |
+          npm i openapi-merge-cli
+          npm i @openapitools/openapi-generator-cli
```

Comment:
We can create a package.json file and also specify the packages versions to avoid potential issues if there are breaking changes in these two packages in the future.